src/pages/our-work.js

```diff
@@ -13,6 +13,22 @@ import Hr from '../components/Common/Hr'
 import Head from '../components/Common/Head'
 import CaseStudy from '../components/OurWork/CaseStudy'
 
+// 'Thomas-Cook` slug is uppercased in Contentful
+const caseStudiesOrder = [
+  'doctorlink',
+  'central-working',
+  'learnerbly',
+  'trainline',
+  'the-economist',
+  'kingfisher',
+  'Thomas-Cook',
+  'joyent',
+  'canon'
+]
```

Comment:
I thought we agreed that the sorting shouldn't be in the code? The ordering should be controlled by contentful and the code should handle fallbacks.

Reply:
I thought it would make sense too. It's ideal but I didn't see it in the ticket so worked in the code. If we agree on something like that it should be in the ticket description.

The Contentful API gives us `publishedAt`, `createdAt` and `updatedAt` which I looked at and didn't manage to make it work. I can look into it again.